src/zope/app/apidoc/static.py

```diff
@@ -138,7 +138,7 @@ def setUserAndPassword(self, user, pw):
         self.addHeader("Authorization", 'Basic ' + encoded)
 
     @classmethod
-    def begin(cls):
+    def begin(cls, *args):
```

Comment:
`OnlineBrowser` and `PublisherBrowser` need to have the same API, especially for `begin`.  The test cases may not adequately cover this fact...